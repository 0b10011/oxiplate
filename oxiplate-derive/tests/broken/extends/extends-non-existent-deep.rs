use oxiplate_derive::{Oxiplate};

#[derive(Oxiplate)]
#[oxiplate = "extends-missing-deep.html.oxip"]
struct Data;

fn main() {
    assert_eq!("", format!("{}", Data));
}
