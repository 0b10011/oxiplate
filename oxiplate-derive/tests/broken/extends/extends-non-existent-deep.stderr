error: Path does not exist: `$DIR/templates/extends-non-existent.html.oxip`
 --> templates/extends-missing.html.oxip
  |
  | {% extends "extends-non-existent.html.oxip" %}
  |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
